kovri.i2p/_i18n/fr/contributing.md

```diff
@@ -78,76 +79,96 @@ C4 is meant to provide a reusable optimal collaboration model for open source so
-- A patch to a stabilization project declared "stable" SHALL be accompanied by a reproducible test case.
-
-### Evolution of Public Contracts
-
-- All Public Contracts (APIs or protocols) SHALL be documented.
-- All Public Contracts SHOULD have space for extensibility and experimentation.
-- A patch that modifies a stable Public Contract SHOULD not break existing applications unless there is overriding consensus on the value of doing this.
-- A patch that introduces new features to a Public Contract SHOULD do so using new names.
-- Old names SHOULD be deprecated in a systematic fashion by marking new names as "experimental" until they are stable, then marking the old names as "deprecated".
-- When sufficient time has passed, old deprecated names SHOULD be marked "legacy" and eventually removed.
-- Old names SHALL NOT be reused by new features.
-- When old names are removed, their implementations MUST provoke an exception (assertion) if used by applications.
-
-### Project Administration
+- Les mainteneurs and les Contributeurs ONT un compte Plateforme and doivent utiliser leur vrais noms ou un alias habituel.
+- Un patch EST une réponse minîme et précise à précisement un problème identifié et défini.
+- Un patch ADHERE au code de style du projet si celui-ci est défini.
+- Un patch ADHERE à l'"Evolution de contrats publiques" défini ci-dessous.
+- Un patch n'INCLUT PAS du code non-trivial d'autres projets à moins que le Contributeur est l'auteur original de ce code.
+- Un patch COMPILE proprement and réussit les tests déstinés à celui-ci pour au moins la platerforme cible principale.
+- Un message de commit d'un patch doit consister en un résumé bref (moins de 50 caractères) des changements, pouvant être suivi d'une ligne blanche et ensuite d'un descriptif plus détaillé.
+- Un "Patch correcte" remplit les conditions énnoncées ci-dessus.
+
+### Procédure de développement
+
+- Les changements du projet SONT GOURVERNÉS par le principe d'identifier précisement les problèmes et d'y appliquer des solutions minimales et précises.
+- Pour proposer des changements, un utilisateur doit créer une issue dans le tracker de problèmes du projet Plateforme.
+- L'utilisateur ou le Contributeur doit écrire l'issue en décrivant le problème observé.
+- L'utilisateur ou le Contributeur doit estimer la précision de leur observation et la valeur de la résolution de leur problème.
+- Les utilisateurs ne CRÉENT PAS de requêtes de fonctionnalités, idées, suggestions ou quelconque solutions aux problèmes qui ne sont pas explicitement documentés et founis.
+- Ainsi, la sortie finale du projet EST une liste de problèmes observés puis fixés.
+- Pour travailler sur un problème, un Contributeur FORK le répertoire du projet and travaille ensuite sur leur répertoire fork.
+- Pour soumettre un patch, un Contributeur CRÉE une requête Plateforme pour le projet.
+- Un contributeur ne COMMIT PAS directement les changements sur le projet.
```

Comment:
"commiter" or "commit" ? because @anonimal specified to let in english as long as  we can